### PR TITLE
improve data connectors documentation

### PR DESCRIPTION
**Pull request by Aminelouati**

### CODE REVIEW

File: src/docs/user/references/DataConnectors.adoc [hunk 1]
```diff
@@ -7,127 +7,212 @@ We begin by presenting FTP and SFTP connectors then, URL connector.
...
 ==== FTP and SFTP connectors
 
-*Task Variables:*
+*Variables:*
 The FTP and SFTP connectors share the same list of variables. Consequently, we describe them in the following table using a unique notation.
 `<PROTOCOL>` can take one of the following values: {`FTP`, `SFTP`}
 
-.FTP/SFTP_Connector_Task variables
-[cols="2,5,2"]
+.FTP/SFTP_Connector variables
```
Comment (amouhoub) [inline src/docs/user/references/DataConnectors.adoc]: You don't need the _ (underscore) as it is not part of workflow/task names

File: src/docs/user/references/DataConnectors.adoc [hunk 2]
```diff
@@ -7,127 +7,212 @@ We begin by presenting FTP and SFTP connectors then, URL connector.
...
 Currently, we have connectors for MongoDB and Cassandra.
 
-*Task Variables:*
+*Variables:*
 
 `<NoSQL_NAME>` can take one of the following values: {`CASSANDRA`, `MONGODB`}
 
-.NoSQL_Connector_Task variables
-[cols="2,5,2"]
+.NoSQL_Connector variables
```
Comment (amouhoub) [inline src/docs/user/references/DataConnectors.adoc]: same comment as above (underscore)

File: src/docs/user/references/DataConnectors.adoc (hunk 1 above)
Comment (Aminelouati) [reply src/docs/user/references/DataConnectors.adoc]: U r right.

File: src/docs/user/references/DataConnectors.adoc [hunk 3]
```diff
@@ -7,127 +7,212 @@ We begin by presenting FTP and SFTP connectors then, URL connector.
...
 Currently, we have connectors for Mysql, Oracle, Postgres and Sql server.
 
-*Task Variables:*
+*Variables:*
 All SQL connectors share the same list of variables. Consequently, we describe them in the following table using a unique notation.
 `<RDBMS_NAME>` can take one of the following values: {`POSTGRES`, `ORACLE`, `SQL_SERVER`, `MYSQL`}
 
-.SQL_Connector_Task variables
-[cols="2,5,2"]
+.SQL Connector_Task variables
```
Comment (MaelAudren) [inline src/docs/user/references/DataConnectors.adoc]: Is it intended to keep task here when removing others ?

Comment (Aminelouati) [reply src/docs/user/references/DataConnectors.adoc]: Not at all. It is a mistake. Thank u for the observation.

**Pull request merged**